src/hypothesis/internal/conjecture/utils.py

```diff
@@ -101,6 +103,21 @@ def centered_integer_range(data, lower, upper, center):
     )
 
 
+def check_sample(values):
+    if isinstance(values, Sequence):
+        return tuple(values)
+    raise InvalidArgument(
```

Comment:
As a general rule I like error messages to contain the value that triggered the error.